tests/python/shared/fixtures/init.py

```diff
@@ -267,38 +258,44 @@ def get_server_image_tag():
     return f"cvat/server:{os.environ.get('CVAT_VERSION', 'dev')}"
 
 
-def start_services(rebuild=False):
+def docker_compose(dc_files, cvat_root_dir):
+    return [
+        "docker-compose",
```

Comment:
@yasakova-anastasia You switched back from Docker Compose V2 to V1; was that intentional? If not, could you revert it?